docs/about/license.rst

```diff
@@ -0,0 +1,4 @@
+License
```

Comment:
Actually, for license, I prefer to get rid of the "about" folder.  I'm ok with license.rst to sit in docs/license.rst.

Reply:
done